app/models/concerns/arclight/solr_document.rb

```diff
@@ -107,7 +103,7 @@ def level
     end
 
     def collection?
-      level.parameterize == 'collection'
+      level&.parameterize == 'collection'
```

Comment:
why wouldn't this have a level?

Reply:
It's probably bad test data, but it's not impossible 🤷‍♂️